ggshield/core/config/auth_config.py

```diff
@@ -69,6 +70,7 @@ def prepare_auth_config_dict_for_parse(data: Dict[str, Any]) -> Dict[str, Any]:
 
     Stop if there is not exactly one account.
     """
+    replace_in_keys(data, "-", "_")
```

Comment:
Can you add a line in the docstring saying that for compatibility issues we turn `-` chars in keys into `_`?

Reply:
Done!